Makefile

```diff
@@ -84,6 +84,10 @@ staticdeps:
 	pip install -t kolibri/dist -r $(REQUIREMENTS)
 	python install_cexts.py --file $(REQUIREMENTS_CEXT) # pip install c extensions
 	rm -r kolibri/dist/*.dist-info  # pip installs from PyPI will complain if we have more than one dist-info directory.
+	# This expression checks that everything in kolibri/dist has an __init__.py
+	# To prevent namespaced packages from suddenly showing up
+	# https://github.com/learningequality/kolibri/pull/2972
+	! find kolibri/dist -mindepth 1 -maxdepth 1 -type d -not -name cext -exec ls {}/__init__.py \; 2>&1 | grep  "cannot access"
```

Comment:
I checked on my local machine (mac os x), the command `find testdist -mindepth 1 -maxdepth 1 -type d -not -name cext -exec ls {}/__init__.py \; 2>&1` will have the result: `ls: testdist/jaraco/__init__.py: No such file or directory`, which doesn't contain "cannot access". Do you think it will be better to change the grep keywords? If we are only using buildkite to run the script, then I think it should be fine, since we are running inside Ubuntu. 
Another question I have is about the exclamation mark in the beginning of the command. It seems that the results are the same with and without the exclamation mark. I'm not really sure about the use of this. Do you mind explaining on this a bit? Thank you!

Reply:
Also, do you think we should fail buildkite if we find a namespace package while building installers? If I understand correctly, currently this command prints a statement to indicate the existence of namespace packages, but we might miss this statement, since the buildkite logs are usually super long? I remember I had a problem with the c extension builds but didn't catch the problem for a long time, because I didn't check the logs of buildkite. Not sure if this would be the case here though... @benjaoming Do you have any thoughts? Thanks!

Reply:
Oh okay :) I'll fix that so at least it works on Mac, too -- I wasn't *that* interested in non-Buildkite platforms tbh, but now that it's tested on OSX, we might as well fix the expression, can just make it detect "No such file". Thanks for checking!

> Also, do you think we should fail buildkite if we find a namespace package while building installers? If I understand correctly, currently this command prints a statement to indicate the existence of namespace packages, but we might miss this statement, since the buildkite logs are usually super long?

I wonder what happened here, I'll figure out a solution.. it's of course supposed to fail only when this line exists.

Reply:
> I wonder what happened here, I'll figure out a solution.. it's of course supposed to fail only when this line exists.

This part works, though, because the exclamation mark `!` creates a non-zero return code. Example:

```
✗ make testdoh 
# This expression checks that everything in kolibri/dist has an __init__.py
# To prevent namespaced packages from suddenly showing up
# https://github.com/learningequality/kolibri/pull/2972
! find kolibri/dist -mindepth 1 -maxdepth 1 -type d -not -name cext -not -name py2only -exec ls {}/__init__.py \; 2>&1 | grep  "No such file"
ls: cannot access 'kolibri/dist/__pycache__/__init__.py': No such file or directory
Makefile:89: recipe for target 'testdoh' failed
make: *** [testdoh] Error 1
```

Reply:
Changed to match for `No such file`